v2/client.go

```diff
@@ -535,15 +531,6 @@ type ClientOptions struct {
 	// connect.
 	Brokers []string
 
-	// MaxConnsPerBroker is the maximum number of connections to pool for a
-	// given broker in the cluster. The default is 2.
-	MaxConnsPerBroker int
-
-	// KeepAliveTime is the amount of time a pooled connection can be idle
-	// before it is closed and removed from the pool. The default is 30
-	// seconds.
-	KeepAliveTime time.Duration
```

Comment:
Even though they no longer do anything, I don't think we can remove this code since it is a breaking code change and we don't want to break existing users. What we should probably do is mark it as deprecated/no longer used and then remove it in the next major version.

Reply:
Fixed in ce8ede5.